java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/IndexField.java

```diff
@@ -46,7 +46,7 @@
 
 import java.util.Objects;
 
-// typedef: _types.mapping.IndexField
+// typedef: _types.mapping.IndexSetting
```

Comment:
How does this work? What does it do?

Reply:
This was done by mistake, I've reverted the change